polkadot/runtime/westend/src/lib.rs

```diff
@@ -1841,6 +1855,7 @@ pub mod migrations {
 		parachains_shared::migration::MigrateToV1<Runtime>,
 		parachains_scheduler::migration::MigrateV2ToV3<Runtime>,
 		pallet_staking::migrations::v16::MigrateV15ToV16<Runtime>,
+		pallet_staking::migrations::v17::MigrateV16ToV17<Runtime>,
```

Comment:
You can check the logs for this migration in this CI job: 

https://github.com/paritytech/polkadot-sdk/actions/runs/12412281493/job/34651846644?pr=6445

there seems to be some error-ish logs in there: 

`💸 Migration failed for ClaimedRewards from v16 to v17.`

Reply:
Solved with `force_from(...)` for `WeakBoundedVec`. I noticed that old individual validator exposures have up to 205 pages, while we would expect them to be at most 20 in the future @Ank4n 
What do you think is the best approach: keep 20 as a limit for `MaxRewardPagesPerValidator` for future items and force old items in a `WeakBoundedVec` or increase the `MaxRewardPagesPerValidator` limit to something like 250 to accomodate old items?